Sources/web3swift/EthereumAPICalls/Ethereum/IEth.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Created by Yaroslav Yashin.
+//  Copyright © 2022 Yaroslav Yashin. All rights reserved.
+//
+
+import Foundation
+import BigInt
+import Core
+
+public protocol IEth {
```

Comment:
I need an explanation of the purpose of that type. And which one have been replaced with it? Means I'll continue to review this given bit of PR afterwards.